gapic/src/main/com/google/gapid/views/PipelineView.java

```diff
@@ -364,6 +364,14 @@ private Composite createStage(Composite parent, API.Stage currentStage) {
 
             Label keyLabel = withLayoutData( createBoldLabel(keyComposite, kvp.getName() + (kvp.getDynamic() ? "*:" : ":")),
                 new GridData(SWT.RIGHT, SWT.CENTER, true, true));
+            
+            if (!kvp.getDependee().equals("")) {
+              keyLabel.setToolTipText((kvp.getActive() ? "Activated" : "Deactivated") + " by " + kvp.getDependee());
+              
+              if (!kvp.getActive()) {
+                keyLabel.setForeground(getDisplay().getSystemColor(SWT.COLOR_TITLE_INACTIVE_FOREGROUND));
```

Comment:
This does not work on Windows, the color looks unchanged.

Reply:
Do they change with setEnabled()?

Reply:
Yes

Reply:
I went with setEnabled() to be safe, but with tables I went with DARK_GRAY. If the system doesn't display tables as expected, then I strongly suggest to keep it but add a message about deactivation in the group title like we do with dynamic tables.

Reply:
That's not a good solution as a fixed color does not play well with different color themes. Also under Linux currently the label background color seems to be slightly different than the composite background.
Maybe we just keep it for now until we have discussed a general solution for the color themes.

Reply:
I can't use setEnabled() with tables. At least on Linux, it doesn't gray out the cells.

Reply:
There is also not coloring the table cells at all and instead using the group title to say it is disabled like we do when table data is set dynamically

Reply:
Ok, we keep it for now.